Avoiding po4a mishaps with lines starting with a "*" (smoe:documentation_20250309) (#3373)
LinuxCNC/linuxcnc: docs/src/man/man1/halmeter.1.adoc

== DESCRIPTION

The tool *halmeter* is used to observe HAL (Hardware Abstraction Layer) pins,
signals, or parameters. It serves the same purpose as a multimeter does

> when working on physical systems. It is an application in its own right and

Reviewer: ```suggestion
when working on physical systems. It is an self-contained application and
```
